portfolio/src/css/Info.css

```diff
@@ -1,134 +0,0 @@
-.info-container {
-    background: linear-gradient(to bottom, #2c333e 63%, #020204 95%);
```

Comment:
if you use these colors elsewhere, one pattern I would suggest is a global CSS color pattern https://github.com/uwigem/wiki2019/blob/master/src/index.css